docs/web-analytics.md

```diff
@@ -5,11 +5,11 @@ sidebar_label: Web Analytics
 ---
 
```

Comment:
Instead of using the code `google_analytics` or `google_tag_manager`, I've a link to each of the sections using a markdown anchor. And also reviewed the text.
Now I remember that I didn't include any text about how a developer should include frontend code when he wants to support a newer web analytics provider.